Assets/Scripts/LookedAtInteractiveDisplayText.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+/// <summary>
+/// This UI Text displays info about the current looked at IInteractive Element.
+/// The text should be hidden if player is not looking at an interactive.
+/// </summary>
+public class LookedAtInteractiveDisplayText : MonoBehaviour
+{
+    private IInteractive lookedAtInteractive;
+    private Text displayText;
+
+    private void Awake()
+    {
+        displayText = GetComponent<Text>();
+        UpdateDisplayText();
+    }
+
+    private void UpdateDisplayText()
```

Comment:
I'd start commenting each individual method.

Reply:
Started and Finished.